dev-infrastructure/Makefile

```diff
@@ -247,7 +247,15 @@ svc.aks.kubeconfigfile:
 	@echo ${SVC_KUBECONFIG_FILE}
 .PHONY: svc.aks.kubeconfigfile
 
-svc.init: region svc svc.aks.admin-access svc.aks.kubeconfig metrics-infra svc.enable-aks-metrics
+svc.oidc.storage.permissions:
+	STORAGEACCOUNTID=$(shell az storage account show -n ${OIDC_STORAGE_ACCOUNT} -g ${SVC_RESOURCEGROUP} --query id -o tsv) && \
+	az role assignment create \
+	--role "Storage Blob Data Contributor" \
+	--assignee ${PRINCIPAL_ID} \
+	--scope "$${STORAGEACCOUNTID}"
+.PHONY: svc.oidc.storage.permissions
+
+svc.init: region svc svc.aks.admin-access svc.aks.kubeconfig metrics-infra svc.enable-aks-metrics svc.oidc.storage.permissions
```

Comment:
I understand that in the order that's been specified, when the svc.oidc.storage.permissions target runs there's already been another target executed that created the storage account itself right?

I also understand that PRINCIPAL_ID in this context is the id of the azure user running the commands. Usually the RH user (msoriano@redhat.com) in my case. Is that correct?

If that's the case, then the MR looks good to me